docs/03-endpoints/api-admin/projects.md

```diff
@@ -814,6 +815,49 @@ Example response:
 }
 ```
 
+### Set Restricted View Settings
+
+Both routes take String parameter which sets restricted view size in one of two formats: as an image dimensions or a 
+percentage. The dimensions pattern looks like: `!X,X`, where X is the number representing scaled image dimensions in
+a square, so that the width and height of the returned image are not greater than the requested value.
+Example: `!512,512` means the image's bigger side will be set to 512 pixels, setting the other side respectively to
+image aspect ratio. The percentage pattern looks like: `pct:X`, where X is the number between 1-100 representing the
+percentage the image will be scaled to. Example: `pct:1` means the image will be scaled to 1% of the original image
+size. **Both routes are implemented in `ZIO-HTTP`, thus can be reached only under port `5555`.**
```

Comment:
> Both routes are implemented in `ZIO-HTTP`

Using zio an implementation detail and can be omitted from the end users documentation, also it is likely that this detail changes but the route will still remain.

> , thus can be reached only under port `5555`.
This is hidden from the user by our load balancer. 

Come to think of it:
`GET /admin/projects/iri/{iri}/RestrictedViewSettings` is on port 3333
`POST /admin/projects/iri/{iri}/RestrictedViewSettings` is on port 5555

The same is true for the `/admin/projects/shortcode/{shortcode}/RestrictedViewSettings` routes. Have you clarified with infrastructure that we can expose both routes on the same port with our loadbalancer.

Reply:
It is an implementation detail, but it doesn't hurt to highlight it for us, because all other routes have both implementations.

`GET` routes are exposed on both ports and `POST` routes are exposed to `5555` only. Thought this could be set on the APP side - @SamuelBoerlin what do you think?

Reply:
The `docs/03-endpoints/api-admin/projects.md` is part of the official end user documentation on docs.dasch.swiss. 
The use of zio will not be helpful for our enduser and might possibly be confusing. It is worthy to point out that is is only available on the 5555 port.

> "but it doesn't hurt to highlight it for us"

We have the "is using zio" information in the code. Shouldn't this be sufficient?

Reply:
> It is an implementation detail, but it doesn't hurt to highlight it for us, because all other routes have both implementations.
> 
> `GET` routes are exposed on both ports and `POST` routes are exposed to `5555` only. Thought this could be set on the APP side - @SamuelBoerlin what do you think?

We don't set port 5555 in the app config.  
The DSP APP we host on [app.dasch.swiss](app.dasch.swiss) prepends /zio/ to ZIO routes and those routes are then internally (i.e. transparent to the end user) redirected to port 5555 by Traefik (and non-ZIO to 3333).  
So, there are two cases that might be relevant for end users of the API depending on whether we talk about [api.dasch.swiss](https://api.dasch.swiss) or DSP-API in general:
1) On [api.dasch.swiss](https://api.dasch.swiss) all routes are exposed over port 443/https, so the ZIO routes are not accessed via port 5555 but instead over port 443 and by prefixing routes with /zio/ (e.g.: https://api.dasch.swiss/zio/admin/projects -> port 5555 vs https://api.dasch.swiss/admin/projects -> port 3333)
2) If you self host DSP API then you access the ZIO routes via port 5555 by default (e.g.: http://localhost:5555/admin/projects vs http://localhost:3333/admin/projects)

Reply:
Thanks @SamuelBoerlin for the explanation and confirmation!
I simply removed that sentence from docs.